Software/WinIOT/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# BrickPi3
+## Windows 10 IoT Core implementation for the BrickPi3.  These libraries can be run with the Raspberry Pi 2 or Raspberry Pi 3.
```

Comment:
Use "# Windows 10 IoT Core implementation for the BrickPi3" as the header. Alternatively shorten it to something like "# Windows 10 IoT Core" since the context of this readme is already BrickPi3 software.

Take out ".  These libraries can be run with the Raspberry Pi 2 or Raspberry Pi 3.", as that information is listed in the BrickPi3 WinIOT repo readme, and could be subject to change (e.g. when RPi 4 is released, or when WinIoT supports other versions of RPi). The RPi2/3 limitation/flexibility is a minor point, not relevant in the context of this readme.